app/controllers/candidate_interface/course_choices/add_another_course_controller.rb

```diff
@@ -0,0 +1,28 @@
+module CandidateInterface
+  module CourseChoices
+    class AddAnotherCourseController < BaseController
+      def ask
```

Comment:
It looks like this `ask`/`decide` pattern is common to a few of the new simplified controllers. Do you think a naming convention like you've introduced here could be applied to some of the others?

At first I wondered whether standard 'RESTful' action names like `index` and `create` might make sense now that each controller is concerned with just one resource/thing.

Reply:
> Do you think a naming convention like you've introduced here could be applied to some of the others?

Not sure! I think it makes sense for those screens where the candidate makes a binary choice about where they go next. Not sure how well it applies to the others in this module

>At first I wondered whether standard 'RESTful' action names like index and create might make sense now that each controller is concerned with just one resource/thing.

Was thinking the same. I'll add a commit that covers this in my followup PR so that we can see what it looks like.